### ISSUE TITLE
Aggregate network throughput on top bar

### ISSUE THREAD

**Issue by SanpuiRonak** [corecoding/Vitals#198]
**Aggregate network throughput on top bar**
It would be convenient and also occupy less space on the gnome top bar if we could put a aggregate network throughput on top bar

Thanks :-)

**Comment by corecoding** [comment]
Can you explain this in a bit more detail? What do you mean by aggregate? Do you mean a double line upload/download stats?

**Comment by SanpuiRonak** [comment]
yup summation of upload+download which can be put on the top bar
